com.unity.render-pipelines.universal/ShaderLibrary/UnityGBuffer.hlsl

```diff
@@ -134,7 +134,7 @@ FragmentOutput SurfaceDataToGbuffer(SurfaceData surfaceData, InputData inputData
     output.GBuffer0 = half4(surfaceData.albedo.rgb, PackMaterialFlags(materialFlags));   // albedo          albedo          albedo          materialFlags   (sRGB rendertarget)
     output.GBuffer1 = half4(surfaceData.specular.rgb, surfaceData.occlusion);            // specular        specular        specular        occlusion
     output.GBuffer2 = half4(packedNormalWS, surfaceData.smoothness);                     // encoded-normal  encoded-normal  encoded-normal  smoothness
-    output.GBuffer3 = half4(globalIllumination, 1);                                      // GI              GI              GI              [optional: see OutputAlpha()] (lighting buffer)
+    output.GBuffer3 = half4(globalIllumination, 1);                                      // GI              GI              GI              unused          (lighting buffer)
```

Comment:
I actually removed this because I did look at OutputAlpha() based on the comment and I didn't find anything about how it interacted with the GBuffer. Not sure exactly what the intention was there.

My changes do actually affect the output alpha in some cases, but none of those cases should involve GBuffers. AlphaToMask requires MSAA so my changes are supposed to be limited to forward rendering only.